eureka-client/src/test/java/com/netflix/discovery/util/EurekaEntityFunctionsTests.java

```diff
@@ -0,0 +1,356 @@
+package com.netflix.discovery.util;
+
+import com.netflix.appinfo.InstanceInfo;
+import com.netflix.discovery.shared.Application;
+import com.netflix.discovery.shared.Applications;
+import org.junit.Assert;
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.HashSet;
+
+public class EurekaEntityFunctionsTests {
+
+    @Test
+    public void testSelectApplicationNames() {
+        Applications applications = new Applications();
+        applications.addApplication(new Application("foo"));
+        applications.addApplication(new Application("bar"));
+
+        HashSet<String> strings = new HashSet<>();
+        strings.add("bar");
+        strings.add("foo");
+        Assert.assertEquals(strings,
+                EurekaEntityFunctions.selectApplicationNames(applications));
+    }
+
+    @Test
+    public void testSelectInstancesMappedById() {
+        InstanceInfo instanceInfo = PowerMockito.mock(InstanceInfo.class);
+        PowerMockito.when(instanceInfo.getId()).thenReturn("foo");
+
+        Application application = new Application();
+        application.addInstance(instanceInfo);
+
+        HashMap<String, InstanceInfo> hashMap = new HashMap<>();
+        hashMap.put("foo", instanceInfo);
+        Assert.assertEquals(hashMap,
+                EurekaEntityFunctions.selectInstancesMappedById(application));
+    }
+
+    @Test
+    public void testSelectInstance() {
+        InstanceInfo instanceInfo = PowerMockito.mock(InstanceInfo.class);
+        PowerMockito.when(instanceInfo.getId()).thenReturn("foo");
+        Application application = new Application("foo");
+        application.addInstance(instanceInfo);
+
+        ArrayList<Application> applicationArrayList = new ArrayList<>();
+        applicationArrayList.add(application);
+        Applications applications = new Applications(
+                "appsHashCode", 1559658285l, applicationArrayList);
+        applications.addApplication(application);
+
+        Assert.assertNull(EurekaEntityFunctions
+                .selectInstance(new Applications(), "foo"));
```

Comment:
I would try to avoid breaking lines in the middle of the qualified method name (this and other examples)
In this case, the alternative might be to break at `.selectInstance(`

Reply:
done